server/src/main/java/org/opensearch/extensions/action/IssueServiceAccountResponse.java

```diff
@@ -0,0 +1,81 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ */
+
+package org.opensearch.extensions.action;
+
+import org.opensearch.core.common.io.stream.StreamInput;
+import org.opensearch.core.common.io.stream.StreamOutput;
+import org.opensearch.core.transport.TransportResponse;
+
+import java.io.IOException;
+import java.util.Objects;
+
+/**
+ * This class represents a Transport Request for issuing a service account to an extension.
+ */
+public class IssueServiceAccountResponse extends TransportResponse {
+
+    private String name;
+    private String serviceAccountString;
```

Comment:
nit: avoid String in variable names. The datatype gives it away that this is a string. 

Since this is the response object, what is the value of including the token on the response?

Reply:
Oh shoot, yeah I don't know what I was thinking for the variable name....

For the response, I was thinking it would be good to have based on the response objects from what I saw in the `InitializeExtensionResponse` file where there is this:


```
    private String name;
    private List<String> implementedInterfaces;

    public InitializeExtensionResponse(String name, List<String> implementedInterfaces) {
        this.name = name;
        this.implementedInterfaces = implementedInterfaces;
    }

    public InitializeExtensionResponse(StreamInput in) throws IOException {
        name = in.readString();
        this.implementedInterfaces = Arrays.asList(in.readStringArray());
    }

    @Override
    public void writeTo(StreamOutput out) throws IOException {
        out.writeString(name);
        out.writeStringArray(implementedInterfaces.toArray(new String[0]));
    }

    /**
     * @return the node that is currently leading, according to the responding node.
     */

    public String getName() {
        return this.name;
    }

    /**
     * @return interfaces implemented by an extension
     */

    public List<String> getImplementedInterfaces() {
        return implementedInterfaces;
    }

    @Override
    public String toString() {
        return "InitializeExtensionResponse{" + "name = " + name + " , " + "implementedInterfaces = " + implementedInterfaces + "}";
    }

    @Override
```
It seemed like the authors were being explicit as to what extension was made and with what interfaces so I figured I would provide explicit info as well.

I can remove it if you think that is better.

Reply:
What would be useful for the response? Can the extension report back if the operation was successful or not? What happens if the transport action times out? (This scenario doesn't need to be handled in this PR, but could represent a real scenario) Would the action be retried if unsuccessful?

Reply:
I think we just need to know that the extension received the service account token. I am not sure that anything other than that is required. 

For retrying and time outs, I would expect extension installation to fail probably but that will likely be a separate PR if we want something more sophisticated than just throwing the same type of error that the InitializeExtensionAction does.